tools/RunSigningJob.ps1

```diff
@@ -1,16 +1,22 @@
 $isPr = Test-Path env:APPVEYOR_PULL_REQUEST_NUMBER
 $directoryPath = Split-Path $MyInvocation.MyCommand.Path -Parent
 
-if (-not $isPr) {
-  Compress-Archive $directoryPath\..\buildoutput\* $directoryPath\..\buildoutput\tosign.zip
+if (-not $isPr -or $env:ForceArtifacts -eq "1") {
+  Compress-Archive -Force $directoryPath\..\..\..\buildoutput\* $directoryPath\..\..\..\buildoutput\tosign.zip
+  Write-Host "Signing payload created at " $directoryPath\..\..\..\buildoutput\tosign.zip
 
   if ($env:SkipAssemblySigning -eq "true") {
     "Assembly signing disabled. Skipping signing process."
     exit 0;
   }
 
+  if ($env:FILES_ACCOUNT_NAME -eq $null -or $env:FILES_ACCOUNT_KEY -eq $null) {
```

Comment:
When debugging locally the script would get to here then fail because these vars aren't present. So handling this more gracefully.